ci-bench/src/main.rs

```diff
@@ -116,8 +117,8 @@ fn main() -> anyhow::Result<()> {
                 .get(index as usize)
                 .ok_or(anyhow::anyhow!("Benchmark not found: {index}"))?;
 
-            let mut stdin = io::stdin().lock();
-            let mut stdout = io::stdout().lock();
+            let mut stdin = util::UnbufferedStdinReader;
```

Comment:
if i've understood this correctly, it would be equivalent to do `let mut stdin = unsafe { File::from_raw_fd(0) };`

Reply:
I like that! The only drawback I see is that each `File` will be closed upon drop, meaning that `stdin` and `stdout` will get closed too. I think that's not a real problem, though, because if we ever want to do use stdio elsewhere, we'll want to do so in an unbuffered way (reusing the files).

(Just pushed the changes use `File`)

Reply:
Could do a `mem::forget()` for the files, I suppose?